src/rpc.cpp

```diff
@@ -133,6 +134,27 @@ RPCServer::RPCServer() {
         RPCObjectID realm_id = store_object(realm_object);
         return (json){{"result", realm_id}};
     };
+    m_requests["/create_user"] = [this](const json dict) {
```

Comment:
Yes, I think you are right. I am trying out just sending the `Sync.User` object in `create_session` and use it like the other objects.